CONTRIBUTING.md

```diff
@@ -32,16 +32,17 @@ Please keep the following in mind:
 
 - Please run the following commands before submitting your PR:
     * `pub run test` - All the tests for Dart exercises can be run from the top level of the repo with this command
-    * `bin/configlet lint .` - Checks the config.json for formatting issues
+    * `bin/fetch-configlet` - Downloads the latest version of configlet and stores it in the bin/ directory
+    * `bin/configlet lint --track-id dart .` - Checks the config.json for formatting issues
     * `bin/configlet fmt .` - Formats the config.json
     * `bin/check_formatting` - Checks All the Dart files for formatting issues
     * `pub run dart_style:format -l 120 -w .` - Formats all the Dart files
 
-- Once a Pull Request is approved, the contributing branch will be Rebased onto the latest version of our master branch.
+- Once a Pull Request is approved, the contributing branch will be rebased onto the latest version of our master branch.
 
 ## Contributing a New Exercise ###
 
-- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
+- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to [Exercism](https://github.com/exercism/problem-specifications/).
```

Comment:
I recommend change last sentence to:
"Please submit a PR to [exercism/problem-specification], if your exercise is new." 
Or 
"If your exercise is new, please submit a PR to [exercism/problem-specification]."

Reply:
I think they wanted to stress that new exercises must first have a specification in problem-specifications. 

So how about:
> All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises). If your exercise is new, please submit a Pull Request to [exercism](exercism/problem-specifications/).

Reply:
I only meant the last sentence. We can keep the current version of "All Exercism exercises must be defined..."